conanfile.py

```diff
@@ -13,6 +13,7 @@ class SDL2Conan(ConanFile):
     exports_sources = ["CMakeLists.txt", "patches/*"]
     generators = ["cmake", "pkg_config"]
     settings = "os", "arch", "compiler", "build_type"
+    version = "2.0.12"
```

Comment:
Don't add a version here, but provide the version via your build command, e.g. `conan create . sdl2/2.0.12@bincrafters/stable`

It will pick up the source code download information from `conandata.yml` accordingly

Within Bincrafters we have also versioned branches if you are concerned that these new options don't exists in previous SDL versions

Reply:
Thanks for the tip about version :)

For me, I'm only interested in the latest versions of SDL with conan at the moment.